org.lflang/src/org/lflang/generator/c/CCmakeGenerator.java

```diff
@@ -241,12 +241,7 @@ CodeBuilder generateCMakeCode(
         cMakeCode.unindent();
         cMakeCode.pr(")");
         cMakeCode.newLine();
-
-        if (this.targetConfig.platformOptions.platform == Platform.ARDUINO) {
```

Comment:
A question is, could we achieve this outside LFC by passing CMake files to "cmake-include" target property? I guess it is fine to have plug-and-play support for a platform like Arduino, but for other embedded boards I think we cant have all this specific logic inside LFC.